docs/source/markdown/podman-network-update.1.md

```diff
@@ -0,0 +1,25 @@
+% podman-network-update 1
+
+## NAME
+podman\-network-update - Update an existing podman network
+
+## SYNOPSIS
+**podman network update**  [*options*] [*name*]
+
+## DESCRIPTION
+Allows end users to update network scoped DNS resolvers for an existing podman network.
```

Comment:
"Allow changes to existing container networks. At present, only changes to the DNS servers in used by a network is supported"